app/services/incremental_refresh_upload_task_service.rb

```diff
@@ -31,6 +31,14 @@ def response_format
     "tar"
   end
 
+  def task_input
```

Comment:
@hsong-rh Can this be inherited from the full_refresh_upload_task since the only things that change between is the  jobs.

Reply:
No need, keep the logic only in parent class